docs/updating/updating_to_3.3.md

```diff
@@ -0,0 +1,128 @@
+# Updating [[= product_name =]] to v3.3
+
+[[= product_name =]] v3.3 uses [Symfony Flex](https://symfony.com/doc/current/quick_tour/flex_recipes.html).
+When updating from v3.2 to v3.3, you need to follow a special update procedure.
+
+If you are updating from an earlier version, start with updating your installation to v3.2.
+
+First, create an upgrade branch and commit your work.
+
+!!! tip
+
+    It is good practice to make git commits after every step of the update procedure.
+
+## Merge composer.json
+
+Merge the special update branch into your project:
+
+```
+git merge origin/v3.2-to-v3.3-upgrade
+```
+
+This will result in conflicts in `composer.json`.
+
+Bring your `composer.json` inline with the changes from the [website skeleton.](https://github.com/ibexa/website-skeleton/blob/main/composer.json)
```

Comment:
Maybe something like this:
```
This merge brings the changes from our website-skeleton (https://github.com/ibexa/website-skeleton/blob/main/composer.json) into your project.

Here are the guidelines for resolving that conflict:
- all `ezsystems/*` packages should be removed
- (...)
```

All of the things mentioned here happen during the merge (except the bin directory cleanup), they are mentioned here explicitly to help resolve the conflicts/let the user know about the final goal he should reach doing the merge.